code/build.gradle

```diff
@@ -188,10 +188,10 @@ repositories {
 
 node {
     // Version of node to use.
-    version = '18.17.1'
+    version = '16.15.1'
```

Comment:
I commit skriver du upgrade, men du mener egentlig downgrade?

Reply:
Det var upgrade til å starte med, så downgradet backend og frontend for at jitpack skulle kunne bygge :)